aiida/manage/configuration/config.py

```diff
@@ -378,38 +378,52 @@ def store(self):
 
         :return: self
         """
-        import tempfile
         from aiida.common.files import md5_from_filelike, md5_file
+        from .settings import DEFAULT_CONFIG_INDENT_SIZE
 
         # If the filepath of this configuration does not yet exist, simply write it.
         if not os.path.isfile(self.filepath):
-            with open(self.filepath, 'wb') as handle:
-                self._write(handle)
+            self._atomic_write()
             return self
 
         # Otherwise, we write the content to a temporary file and compare its md5 checksum with the current config on
         # disk. When the checksums differ, we first create a backup and only then overwrite the existing file.
         with tempfile.NamedTemporaryFile() as handle:
-            self._write(handle)
+            json.dump(self.dictionary, handle, indent=DEFAULT_CONFIG_INDENT_SIZE)
```

Comment:
is there a reason not to use `self._write` here?

Reply:
`self._write` is no longer there, it has been completely replaced by `_atomic_write`

Reply:
ah!